include/exadg/solvers_and_preconditioners/multigrid/multigrid_preconditioner_base.h

```diff
@@ -292,7 +292,10 @@ class MultigridPreconditionerBase : public PreconditionerBase<Number>
   initialize_smoother(Operator & matrix, unsigned int level);
 
   void
-  initialize_chebyshev_smoother(Operator & matrix, unsigned int level);
+  initialize_chebyshev_smoother_point_jacobi(Operator & matrix, unsigned int level);
+
+  void
+  initialize_chebyshev_smoother_block_jacobi(Operator & matrix, unsigned int level);
```

Comment:
This is also present in the `initialize_smoother` function above, but shouldn't we choose `const` arguments in this function?

```suggestion
  initialize_chebyshev_smoother_point_jacobi(Operator & matrix, unsigned int const level);

  void
  initialize_chebyshev_smoother_block_jacobi(Operator & matrix, unsigned int const level);
```

Reply:
Yes we should, I updated this PR.